docs/dorothy/dorothy-not-loading.md

```diff
@@ -76,3 +76,50 @@ Close the preferences and open a new terminal tab, now Dorothy should be loaded
 4. Save the settings.
 
 5. Use the menu bar `Termina: New Terminal` or the Command Palette `Terminal: Create New Terminal` to open a new terminal. It should now start as a login shell, loading Dorothy as expected. Validate this by running a Dorothy command, like `dorothy theme`, to ensure everything is functioning as intended.
+
+## Debian Gnome
+
+1. Select Edit ▸ Preferences.
+
+2. In the sidebar, click on the + button next to the Profiles label.
+
+3. Enter a name for the new profile. You can change this name later.
+
+4. Click Create to create the new profile.
+
+5. Navigate to command and select 'Run a custom command instead of my shell.'
+
+6. Type in 'bash -l' to use bash as your command (or another shell of your choosing with the login shell mode enabled).
+
+7. Click on the arrow next to the profile name (in the left hand sidebar).
+
+9. Select 'Set as default.'
+
+10. Save the profile and restart your terminal.
+
+Source: https://help.gnome.org/users/gnome-terminal/stable/pref-profiles.html.en
+
+## Debian Konsole (KDE)
+
+Note from KDE.org:
```

Comment:
Not sure I understand where the opinion is. I wrote all the steps after step 4 (and had a source to give credit to the partial copy/paste). I wrote these steps because zsh -l would be used if the user preferred zsh, and I am not familiar with all other shells which may have a different flag for login shell mode. For example, Yash.

It's important to note that if someone is configuring Debian to use a shell with login mode, that they get it right in this field. Just putting 'zsh' in there is not going to do it. The argument to be made is that a user should know what command to put in here, and maybe we'd assume that they're more self reliant. However, I think that defeats the point of having a 'having issues with getting Dorothy' to run page?

If the steps are not needed, I figure 'So what to do when your terminal isn't opening a login shell?' could be answered with 'it depends on your OS/terminal, please consult your documentation.' However, that makes Dorothy less accessible to those that are not experts in shells.

Reply:
Whoops, my commentary here is actually for the following lines, see: https://github.com/bevry/dorothy/pull/262#discussion_r1872480284